src/indicator_cumulative_number.Rmd

```diff
@@ -65,7 +70,139 @@ In order to not consider subspecies, please take care of grouping by `species` (
 
 About point 7 of [#21](https://github.com/trias-project/pipeline/issues/21), please see this [comment](https://github.com/trias-project/pipeline/issues/21#issuecomment-381164874).
 
-## Cumulative number of alien species
+## Number of alien species
 
 see GitHub issue [#20](https://github.com/trias-project/pipeline/issues/20)
 
+```{r determine_presence_per_specieskey_per_year}
+
+#' Create total invasive species indicator plot (Trias)
+#'
+#' @param df data.frame Contains the data as produced by the Trias pipeline,
+#' with minimal colums.
+#' @param start_year_plot int Limit to use as start year of the plot. For scientific
+#' usage, the entire period could be relevant, but for policy purpose, focusing
+#' on a more recent period could be required.
+#' @param x_major_scale_stepsize int On which year interval labels are placed 
+#' on the x axis.
+#' @param x_minor_scale_stepsize int On which year interval minor breaks are
+#' placed on the x axis.
+#' @param facet_column NULL | char The column to use to create additional
+#' facet wrap plots underneath the main graph. When NULL, no facet graph is
+#' included.
+#'
+#' @return ggplot2 object
+#' @export
+#' @importFrom assertthat assert_that
+#' @importFrom assertable assert_colnames
+#' @importFrom dplyr distinct_ %>% filter rowwise do bind_cols group_by_ 
+#' count ungroup
+#' @importFrom tidyr replace_na unnest
+#' @importFrom lubridate year now
+#' @importFrom ggplot2 geom_line aes xlab ylab scale_x_continuous facet_wrap
+#' @importFrom INBOtheme theme_inbo
+#' @importFrom egg ggarrange
+#'
+#' @examples
+indicator_total_year <- function(df, start_year_plot = 1940, 
+                                 x_major_scale_stepsize = 10,
+                                 x_minor_scale_stepsize = 5,
+                                 facet_column = NULL) {
+    
+    # initial input checks
+    assert_that(is.data.frame(df))
+    assert_that(x_major_scale_stepsize >= x_minor_scale_stepsize)
+    assert_colnames(df, c("key", "first_observed", "last_observed"), 
+                    only_colnames = FALSE)
+    
+    # prepare the data set, !recheck when issue #30 cleared out
```

Comment:
I do not know if the recheck is done now issue 30 is done? Should this be emoved?